runtime/gc_vlhgc/VLHGCAccessBarrier.cpp

```diff
@@ -247,47 +250,84 @@ MM_VLHGCAccessBarrier::postStoreClassToClassLoader(J9VMThread *vmThread, J9Class
 	}
 }
 
+void
+MM_VLHGCAccessBarrier::copyArrayCritical(J9VMThread *vmThread, GC_ArrayObjectModel *indexableObjectModel,
+					J9InternalVMFunctions *functions, void **data,
+					J9IndexableObject *arrayObject, jboolean *isCopy)
```

Comment:
Nitpick: I'd rather see a single long line or a single tab indent for the additional parameters